WcaOnRails/Gemfile

```diff
@@ -68,6 +68,7 @@ gem 'daemons'
 gem 'i18n-country-translations'
 gem 'http_accept_language'
 gem 'twitter_cldr'
+gem "jquery-slick-rails"
```

Comment:
Super nitpick, but i think we use single quotes through this file, it would be nice to be consistent.